tests/app/user/test_rest.py

```diff
@@ -731,3 +731,38 @@ def test_get_orgs_and_services_only_shows_users_orgs_and_services(admin_request,
             }
         ]
     }
+
+
+def test_search_for_users_by_email(notify_db, client):
+    user_1 = create_user(email='findel.mestro@foo.com')
+    user_2 = create_user(email='me.ignorra@foo.com')
+    data = json.dumps({"email": "findel"})
+    auth_header = create_authorization_header()
+
+    response = client.post(
+        url_for("user.find_users_by_email"),
+        data=data,
+        headers=[('Content-Type', 'application/json'), auth_header]
+    )
+    users = json.loads(response.get_data(as_text=True))
+
+    assert response.status_code == 200
+    assert len(users['data']) == 1
+    assert users['data'][0]['email_address'] == 'findel.mestro@foo.com'
+
+
+def test_search_for_users_by_email_handles_incorrect_data_format(notify_db, client):
```

Comment:
Is it worth a test where you get an 

1. Empty return? i.e. data = json.dumps({"email": "not.a.user"})
2. A full email i.e. data = json.dumps({"email": "me.ignorra@foo.com})

Reply:
Good call, I will add these tests ☺️